scripts/apps/ingest/directives/IngestUserDashboard.ts

```diff
@@ -27,8 +27,18 @@ export function IngestUserDashboard(api, userList, privileges, moment) {
                     },
                 };
 
-                api.ingest.query(criteria).then((result) => {
-                    scope.ingested_count = result._meta.total;
+                scope.item.content_types.map((types) => {
```

Comment:
not sure with the map here, seems like if you have selected 4 types (text/picture/audio/video) this would run the query 4x